autoware_launch/launch/autoware.launch.xml

```diff
@@ -27,33 +45,34 @@
 
   <!-- Vehicle -->
   <group>
```

Comment:
I guess you can write `if` in the `group` tag. I personally think it looks better.
https://github.com/ros2/launch/blob/cb39d92dd99d821f0aa59b40895303c5b772790b/launch/launch/action.py#L64

```suggestion
  <group if="$(var launch_vehicle)">
```

Reply:
https://github.com/autowarefoundation/autoware_launch/pull/59/commits/45de34b70d82ba22c5cccca34e3892cea9d2f7e8
DONE